### PR TITLE
Update README.md // Made it cringier

### PR DESCRIPTION
**Pull request by harshmathurx**

### CODE REVIEW

**Comment by KonradLinkowski** [comment]
Looks cringy enough for me, but the image is pretty big.
1. Move it to the bottom, make it smaller or make a table with this image and a gif with is under it to make it in one row.
2. Add this image to the image folder and refer to it using a relative path.
image

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
 [![cool](https://img.shields.io/badge/be-cool-purple.svg)](https://shields.io/)
 [![cool](https://img.shields.io/badge/sample-text-green.svg)](https://shields.io/)
+![image](https://user-images.githubusercontent.com/58532371/136007344-824e02f0-8857-4bbb-a9e4-325da0ee0b07.png)
```
Comment (KonradLinkowski) [inline README.md]: Try:
``

Comment (harshmathurx) [reply README.md]: Done

**Review by KonradLinkowski — APPROVED** [review]

**Comment by KonradLinkowski** [comment]
Relates to #1

**Pull request merged**